.drone.starlark

```diff
@@ -409,6 +492,22 @@ config = {
 			],
 			'runCoreTests': True,
 		},
+		'core-cli-acceptance-latest-nightly': {
+			'suites': [
+				'cliTrashbin',
+			],
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
```

Comment:
compared to the section above, this new section runs against `latest` rather than `daily-master-qa`